contracts/stabilizer/PegKeeperV2.vy

```diff
@@ -15,13 +15,20 @@ interface Regulator:
 interface CurvePool:
     def balances(i_coin: uint256) -> uint256: view
     def coins(i: uint256) -> address: view
-    def calc_token_amount(_amounts: uint256[2], _is_deposit: bool) -> uint256: view
-    def add_liquidity(_amounts: uint256[2], _min_mint_amount: uint256) -> uint256: nonpayable
-    def remove_liquidity_imbalance(_amounts: uint256[2], _max_burn_amount: uint256) -> uint256: nonpayable
     def get_virtual_price() -> uint256: view
     def balanceOf(arg0: address) -> uint256: view
     def transfer(_to: address, _value: uint256) -> bool: nonpayable
 
+interface CurvePoolOld:
+    def calc_token_amount(_amounts: uint256[2], _is_deposit: bool) -> uint256: view
+    def add_liquidity(_amounts: uint256[2], _min_mint_amount: uint256) -> uint256: nonpayable
+    def remove_liquidity_imbalance(_amounts: uint256[2], _max_burn_amount: uint256) -> uint256: nonpayable
+
+interface CurvePoolNG:
+    def calc_token_amount(_amounts: DynArray[uint256, 2], _is_deposit: bool) -> uint256: view
```

Comment:
The size of the array is 8, so it should be Dynarray[uint256, 8].

Reply:
PegKeepers work only with 2-coins pools, though good to be precise